### PR TITLE
Fix RSS links

### PR DESCRIPTION
**Pull request by Ksinia**
Params of the RSS subscription URL were messed up. This PR fixes that.

### CODE REVIEW

File: frontend/app/components/comment-form/__subscribe-by-rss/comment-form__subscribe-by-rss.test.tsx [hunk 1]
```diff
@@ -31,9 +31,9 @@ describe('<SubscribeByRSS/>', () => {
     expect(wrapper.find('.comment-form__rss-dropdown__link')).toHaveLength(3);
   });
 
-  it('should have userId in site link', () => {
-    expect(wrapper.find('.comment-form__rss-dropdown__link').at(1).prop('href')).toEqual(
-      createSubscribeUrl('site', '&user=user-1')
+  it('should have userId in Replies link', () => {
```
Comment (akellbl4) [inline frontend/app/components/comment-form/__subscribe-by-rss/comment-form__subscribe-by-rss.test.tsx:34]:
```suggestion
  it('should have userId in replies link', () => {
```

File: frontend/app/components/comment-form/__subscribe-by-rss/comment-form__subscribe-by-rss.test.tsx [hunk 2]
```diff
@@ -31,9 +31,9 @@ describe('<SubscribeByRSS/>', () => {
...
+    expect(wrapper.find('.comment-form__rss-dropdown__link').at(2).prop('href')).toEqual(
```
Comment (akellbl4) [inline frontend/app/components/comment-form/__subscribe-by-rss/comment-form__subscribe-by-rss.test.tsx:35]: I know it was here. But the right way to compare primitive values over `toBe`. `toEqual` was added for compare objects or arrays. Let's fix this ;)
```suggestion
    expect(wrapper.find('.comment-form__rss-dropdown__link').at(2).prop('href')).toBe(
```

**Comment by akellbl4** [comment]
@Ksinia thanks for the contribution, all good. Only a couple minor changes ;)

**Review by akellbl4 — APPROVED** [review]

**Review by umputun — APPROVED** [review]
LGTM, thx

**Pull request merged**